crm/api/doc.py

```diff
@@ -726,3 +727,90 @@ def getCounts(d, doctype):
 		"FCRM Note", filters={"reference_doctype": doctype, "reference_docname": d.get("name")}
 	)
 	return d
+
+
+@frappe.whitelist()
+def getLinkedDocs(doctype, docname):
```

Comment:
Never use camelCase in python code only use snake_case and in js use camelCase not snake_case